lib/solidus_importer/processors/variant.rb

```diff
@@ -29,7 +29,7 @@ def process_variant
           variant.price = @data['Variant Price'] if @data['Variant Price'].present?
 
           # ensure the sku is properly set (for master sku)
-          variant.sku = sku
+          variant.sku = sku if master_variant?
```

Comment:
Why is this now required?  Did add this for better readability?

Reply:
This was causing an error with a spec, so I fixed it on the way 🪄